README.md

```diff
@@ -14,135 +92,49 @@ This repository is currently under heavy development - if you have suggestions o
 
 Stable Release
 ```Shell
-pip install torchao
+pip install torchao --extra-index-url https://download.pytorch.org/whl/test/cu121 # full options are cpu/cu118/cu121/cu124
 ```
 
 Nightly Release
 ```Shell
-pip install --pre torchao-nightly --index-url https://download.pytorch.org/whl/nightly/cpu # CPU only builds
-pip install --pre torchao-nightly --index-url https://download.pytorch.org/whl/nightly/cu118 # CUDA 11.8
-pip install --pre torchao-nightly --index-url https://download.pytorch.org/whl/nightly/cu121 # CUDA 12.1
-pip install --pre torchao-nightly --index-url https://download.pytorch.org/whl/nightly/cu124 # CUDA 12.4
-
+pip install --pre torchao-nightly --index-url https://download.pytorch.org/whl/nightly/cu121 # full options are cpu/cu118/cu121/cu124
 ```
 
-From source
+## Community Contributions
+
+* [jeromeku](https://github.com/jeromeku) has implemented
+    * [GaLore](torchao/prototype/galore/) a drop for the Adam Optimizer that allows you to finetune llama 7b on a single 4090 card with up to 70% speedups relative to eager PyTorch
+    * [DoRA](torchao/prototype/dora) a newer replacement for QLoRA with more promising convergence characteristics
+    * [Fused int4/fp16 Quant Matmul](torchao/prototype/hqq) which is particularly useful for compute bound kernels showing 4x speedups over tinygemm for larger batch sizes such as 512
```

Comment:
tinygemm is mentioned here. Should we link it to a doc section talking about tinygemm, if it exists?

Reply:
Yeah tinygemm is a great idea, will add